wml_test_schedule

```diff
@@ -321,6 +321,7 @@
 0 for_end2_step2
 0 for_end-2
 0 for_end-2_step-2
+0 nested_foreach
```

Comment:
I mixed up the names, will force-push to fix this once the local build has confirmed that I've got it right this time.